language_service/src/signature_help/tests.rs

```diff
@@ -244,7 +244,6 @@ fn last_argument() {
     );
 }
 
-#[ignore = "Parser needs updating to handle `(1,, \"Four\")`"]
```

Comment:
I love that there was already a test for this!

Reply:
Thank @ScottCarda-MS for working ahead ;)